### PR TITLE
ADD HowToUse component

### PR DESCRIPTION
**Pull request by VitaliiDTS**

### CODE REVIEW

File: src/Components/HowToUse_guide/HowToUse_guide.jsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
+import './Styles/HowToUse_guide.css'
+import reg from "../HowToUse_guide/SVG/image 23.svg";
+import join from "../HowToUse_guide/SVG/image 27.svg";
+import create from "../HowToUse_guide/SVG/image 29.svg";
+import earn from "../HowToUse_guide/SVG/image 30.svg";
```
Comment (viktor-kindrat) [inline src/Components/HowToUse_guide/HowToUse_guide.jsx:5]: remove whitespaces from file names. Dont use svg from non-vector images

File: src/Components/HowToUse_user/HowToUse_user.jsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
+import './Styles/HowToUse_user.css'
+import reg from './SVG/image 23.svg'
+import travel from './SVG/image 20.svg'
+import filter from './SVG/image 24.svg'
+import choose from './SVG/image 25.svg'
```
Comment (viktor-kindrat) [inline src/Components/HowToUse_user/HowToUse_user.jsx:5]: remove whitespaces from file names. Dont use svg from non-vector images

File: src/Components/HowToUse_user/Styles/HowToUse_user.scss [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+      }
+
+      .HowToUse__para {
+        @extend %big-plain-text;
+        display: flex;
+        color: $dark;
+      }
+
+      .HowToUse__img {
+        box-shadow: $shadows-black;
```
Comment (viktor-kindrat) [inline src/Components/HowToUse_user/Styles/HowToUse_user.scss:37]: missed width or height value

File: src/Components/HowToUse_user/Styles/HowToUse_user.scss [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+      .HowToUse__img {
+        width: 380px;
+      }
+    }
+
+    .HowToUse__left {
+      padding: $manual-padding;
+
+      .HowToUse__img {
+        width: 380px;
```
Comment (viktor-kindrat) [inline src/Components/HowToUse_user/Styles/HowToUse_user.scss:89]: don't use fixed width

File: src/Components/HowToUse_user/Styles/HowToUse_user.scss [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+      text-align: center;
+      gap: $gap-between-connected-elements;
+
+      .HowToUse__text {
+        align-items: center;
+        gap: $gap-between-closely-placed-elements;
+      }
+
+      .HowToUse__img {
+        width: 380px;
```
Comment (viktor-kindrat) [inline src/Components/HowToUse_user/Styles/HowToUse_user.scss:126]: dont use fixed width

**Review by viktor-kindrat — CHANGES_REQUESTED** [review]
❌ Fixes required

File: src/Components/HowToUse_user/Styles/HowToUse_user.scss (hunk 3 above)
Comment (VitaliiDTS) [reply src/Components/HowToUse_user/Styles/HowToUse_user.scss:38]: fixed

File: src/Components/HowToUse_user/HowToUse_user.jsx (hunk 2 above)
Comment (VitaliiDTS) [reply src/Components/HowToUse_user/HowToUse_user.jsx]: fixed

File: src/Components/HowToUse_guide/HowToUse_guide.jsx (hunk 1 above)
Comment (VitaliiDTS) [reply src/Components/HowToUse_guide/HowToUse_guide.jsx]: fixed

File: src/Components/HowToUse_user/Styles/HowToUse_user.scss (hunk 4 above)
Comment (VitaliiDTS) [reply src/Components/HowToUse_user/Styles/HowToUse_user.scss]: fixed

File: src/Components/HowToUse_user/Styles/HowToUse_user.scss (hunk 5 above)
Comment (VitaliiDTS) [reply src/Components/HowToUse_user/Styles/HowToUse_user.scss]: fixed

File: src/Components/HowToUse_guide/HowToUse_guide.jsx [hunk 6]
```diff
@@ -0,0 +1,64 @@
+import './Styles/HowToUse_guide.css'
+import reg from "./PNG/image23.png";
+import join from "./PNG/image27.png";
+import create from "./PNG/image29.png";
+import earn from "./PNG/image30.png";
```
Comment (viktor-kindrat) [inline src/Components/HowToUse_guide/HowToUse_guide.jsx:5]: convert to webp

File: src/Components/HowToUse_user/HowToUse_user.jsx [hunk 7]
```diff
@@ -0,0 +1,69 @@
+import './Styles/HowToUse_user.css'
+import reg from './PNG/image23.png'
+import travel from './PNG/image20.png'
+import filter from './PNG/image24.png'
+import choose from './PNG/image25.png'
```
Comment (viktor-kindrat) [inline src/Components/HowToUse_user/HowToUse_user.jsx:5]: convert to webp

**Review by viktor-kindrat — COMMENTED** [review]
❌ small fixes

**Review by viktor-kindrat — APPROVED** [review]
✅ Looks well

**Pull request merged**